aws/resource_aws_route53_record.go

```diff
@@ -42,9 +41,12 @@ func resourceAwsRoute53Record() *schema.Resource {
 				Required: true,
 				ForceNew: true,
 				StateFunc: func(v interface{}) string {
-					value := strings.TrimSuffix(v.(string), ".")
```

Comment:
If there are valid other places that can give this resource a trailing period, I guess we may want to walk back on adjusting this particular attribute and allow it to still handle both with and without the trailing period. I'd rather avoid changing the ACM validation record behavior since that could affect interactions with DNS systems outside of AWS.

Reply:
Thank you for updating the comment there, super helpful 👍